File: Folder Python/Smartphone.py
from Elektronik import Elektronik

class Smartphone(Elektronik):
    def __init__(self, kode, nama, harga,
                 merek, garansi, daya,
                 ram, penyimpanan, kamera):
        super().__init__(kode, nama, harga, merek, garansi, daya)
        self.ram = ram
        self.penyimpanan = penyimpanan
        self.kamera = kamera

    def getRam(self):
        return self.ram

    def getPenyimpanan(self):
        return self.penyimpanan

    def getKamera(self):
        return self.kamera

File: Folder Java/Smartphone.java
public class Smartphone extends Elektronik {
    private int ram, penyimpanan, kamera;

    public Smartphone(String kode, String nama, int harga,
                      String merek, String garansi, int daya,
                      int ram, int penyimpanan, int kamera) {
        super(kode, nama, harga, merek, garansi, daya);
        this.ram = ram;
        this.penyimpanan = penyimpanan;
        this.kamera = kamera;
    }

    public int getRam() { return ram; }
    public int getPenyimpanan() { return penyimpanan; }
    public int getKamera() { return kamera; }
}